src/detail/Details.tsx

```diff
@@ -28,11 +31,13 @@ export interface IDetailApiConfig {
 
 export interface IDetailsConfig extends IDetailApiConfig {
     pageTitle?: string;
+    identifiers?: any;
     propertiesConfig: Array<IPropertiesConfig>;
 }
 
-const Details: React.FC = ({ pageTitle, propertiesConfig, detailApiConfig } : IDetailsConfig ) => {
+const Details: React.FC = ({ pageTitle, propertiesConfig, detailApiConfig, identifiers } : IDetailsConfig ) => {
     const [ recordInfo, setRecordInfo ] = useState<IPropertiesConfig[]>( propertiesConfig )
+    // TODO: remove the dynamic-id option from here and use the identifiers prop instead
```

Comment:
I think that it should be the job of the actual parent [either the router or the model] to provide the identifier/s for forms and detail pages. how they [the parent] get the identifiers can be the implementation detail of the parent.